components/teams/team.vue

```diff
@@ -11,18 +11,30 @@
         <i class="material-icons">more_vert</i>
       </button>
       <template slot="popover">
-        <div>
+        <div v-if="team.myRole === 'OWNER'">
           <button class="icon" @click="$emit('edit-team')" v-close-popover>
             <i class="material-icons">create</i>
             <span>{{ $t("edit") }}</span>
           </button>
         </div>
-        <div>
+        <div v-if="team.myRole === 'OWNER'">
           <button class="icon" @click="removeTeam" v-close-popover>
             <i class="material-icons">delete</i>
             <span>{{ $t("delete") }}</span>
           </button>
         </div>
+        <div>
+          <button v-if="team.myRole === 'OWNER' && team.ownersCount == 1" class="icon" @click="exitTeam" v-close-popover disabled>
+            <i class="material-icons">remove</i>
+            <div  v-tooltip.left="$t('disable_exit')">
+              <span>{{ $t("exit") }}</span>
+            </div>
+          </button>
+          <button v-else class="icon" @click="exitTeam" v-close-popover>
+            <i class="material-icons">remove</i>
+            <span>{{ $t("exit") }}</span>
+          </button>
```

Comment:
Instead of two button elements, we can go with an approach like this:

```suggestion
          <button class="icon" @click="exitTeam" v-close-popover :disabled="!(team.myRole === 'OWNER' && team.ownersCount == 1)">
            <i class="material-icons">remove</i>
            <div v-tooltip.left="{ content: team.myRole === 'OWNER' && team.ownersCount == 1 ? null : $t('disable_exit') }">
              <span>{{ $t("exit") }}</span>
            </div>
          </button>
```

It's just a suggestion for better code reusability. However for a disabled button, there's no need for `@click="exitTeam"` action.